src/index.js

```diff
@@ -33,10 +48,45 @@ mongoose
 		console.log("Database connected.");
 	});
 
+var userType = "";
```

Comment:
Never use `var` for anything. It's pretty much soft deprecated in ES6.
(I know some old guides/articles use it, but `var` is terrible, and has no usecase today)